index.html

```diff
@@ -24,148 +24,160 @@
+                                <p>Ruby</p>
+                            </div>
+                            <button type="button">See Project</button>
                         </div>
-                        <button type="button">See Project</button>
+                        <div class="rect"></div>
                     </div>
                 </div>
             </div>
-        </div>
-    </section>
-    <section class="contact" id="Contact-section">
-        <div class="contact-text">
-            <h5 class="chat">Lets's Chat!</h5>
-            <h5 class="mobile">CONTACT US</h5>
-            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
-            </p>
-        </div>
-        <form action="https://formspree.io/f/mayadbao" method="POST">
-            <div class="data">
-                <input type="text" class="contact-element" name="user-f&lname" placeholder="First and Last Name" maxlength="30" required>
+        </section>
+        <section class="contact" id="Contact-section">
+            <div class="contact-text">
+                <h5 class="chat">Lets's Chat!</h5>
+                <h5 class="mobile">CONTACT US</h5>
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+                </p>
             </div>
-            <label for="email"></label>
-            <input type="email" id="email" class="contact-element1" name="_replyto" placeholder="danielowiri4zoe@gmail.com" required><br>
-            <label for="message"></label>
-            <textarea name="message" id="message" class="contact-element2" cols="30" rows="10" placeholder="Write your message here" maxlength="500" required></textarea><br><br>
-            <button type="submit" class="contact-button">Send message</button>
-        </form>
-        <hr class="bottom-line">
-        <footer class="socials" id="send-mail">
-          <img src="./assets/images/social_media.png" alt="social media icons">
-          <p>danielowiri4zoe@email.com</p>
-        </footer>
-      </section>
+            <form action="https://formspree.io/f/mayadbao" method="POST">
+                <div class="data">
+                    <input type="text" class="contact-element" name="user-f&lname" placeholder="First and Last Name" maxlength="30" required>
+                </div>
+                <label for="email"></label>
+                <input type="email" id="email" class="contact-element1" name="_replyto" placeholder="danielowiri4zoe@gmail.com" required><br>
+                <label for="message"></label>
+                <textarea name="message" id="message" class="contact-element2" cols="30" rows="10" placeholder="Write your message here" maxlength="500" required></textarea><br><br>
+                <button type="submit" class="contact-button">Send message</button>
```

Comment:
Please try and stick to the design.
The original design 👇🏽

![Screenshot 2021-07-22 193145](https://user-images.githubusercontent.com/61285247/126692217-801d8a66-4811-4834-be84-69484434a3ff.png)
Your design 👇🏽

![Screenshot 2021-07-22 193054](https://user-images.githubusercontent.com/61285247/126692401-65603e52-e0a2-434e-b261-371ce989e15d.png)
As you can see on the desktop version of the original design the name field is divided into 2 parts but on yours displays what it is supposed to display on mobile view so based on the pictures I have sent you just change the name field and all the placeholders.